### PR TITLE
fix(core-notification): remove bold from success and error state

### PR DESCRIPTION
**Pull request by agorovyi**
## Related issues

See #937 

## Checklist before submitting pull request

- [ ] New code is unit tested
- Commits follow our [Developer Guide](https://tds.telus.com/contributing/developer-guide.html#make-a-commit)
- [ ] For code changes, run `npm run prepr` locally
  - make sure visual and accessibility tests pass

### CODE REVIEW

File: packages/Notification/__tests__/Notification.spec.jsx [hunk 1]
```diff
@@ -44,12 +43,6 @@ describe('<Notification />', () => {
   })
 
   describe('successful variant', () => {
-    it('bolds the content', () => {
```
Comment (jraff) [inline packages/Notification/__tests__/Notification.spec.jsx]: Do we want to test for default `font-weight`?

File: packages/Notification/__tests__/Notification.spec.jsx [hunk 2]
```diff
@@ -58,12 +51,6 @@ describe('<Notification />', () => {
   })
 
   describe('error variant', () => {
-    it('bolds the content', () => {
```
Comment (jraff) [inline packages/Notification/__tests__/Notification.spec.jsx]: Do we want to test for default `font-weight`?

File: packages/Notification/__tests__/Notification.spec.jsx (hunk 1 above)
Comment (agorovyi) [reply packages/Notification/__tests__/Notification.spec.jsx]: Done

File: packages/Notification/__tests__/Notification.spec.jsx (hunk 2 above)
Comment (agorovyi) [reply packages/Notification/__tests__/Notification.spec.jsx]: Done

**Comment by agorovyi** [comment]
> e2e screenshots need to be updated.

Done ✅

**Comment by jraff** [comment]
r+

**Pull request merged**